.github/workflows/main.yaml

```diff
@@ -267,9 +242,10 @@ jobs:
         with:
           r-version: ${{ matrix.r_version }}
       - uses: r-lib/actions/setup-pandoc@v2
-      - name: Install dependencies
-        run: |
-          Rscript -e "options(repos=structure(c(CRAN='https://cloud.r-project.org/'))); install.packages(c('remotes', 'rcmdcheck', 'fs'))" -e "remotes::install_deps(dependencies = TRUE)"
+      - uses: r-lib/actions/setup-r-dependencies@v2
```

Comment:
I'm not sure this will work. I added the win-gpu label to test this workflow in the next commits.

Reply:
Sounds good. It will work, if doesn't hit the lockfile bug in https://github.com/r-lib/pak/issues/358.

Reply:
Yeah, the problem is that this computer doesn't have the same config as in GHA. So for instance, we had to manually configure the default CRAN repository. 
I'd revert that in this PR as there's no easy way to test if this will work correctly.